src/routes/safe/components/Apps/components/AppsList.tsx

```diff
@@ -120,42 +102,72 @@ const AppsList = (): React.ReactElement => {
       </Menu>
       <ContentWrapper>
         <SearchInputCard value={appSearch} onValueChange={(value) => setAppSearch(value.replace(/\s{2,}/g, ' '))} />
-        {noAppsFound && <NoAppsFound query={appSearch} onWalletConnectSearch={() => setAppSearch('WalletConnect')} />}
 
+        {showPinnedApps && (
+          <Collapse
+            title={
+              <Text color="placeHolder" strong size="md">
+                BOOKMARKED APPS
```

Comment:
Btw, @mikheevm please don't use caps. It should be done via CSS (`text-transform: uppercase`).